demo.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from enum import IntEnum
+
+from telegram import ReplyKeyboardMarkup, ReplyKeyboardRemove
+from telegram.ext import CommandHandler, MessageHandler, Filters, ConversationHandler
+
+
+class DemoDecisions(IntEnum):
+    FAVPIC = 1
+    FAMILY = 2
+    AMAZON = 3
+    DURABLE = 4
+    DIFFICULT = 5
+    MICHAEL = 6
+    WINNER = 7
+
+
+yes_no_keyboard = ReplyKeyboardMarkup([["Yes", "No"]])
+filter_yes = Filters.regex("^Yes$")
+filter_no = Filters.regex("^No$")
+
+
+def demo_intro(update, context):
+    text = "Send your favorite picture of Covid!"
```

Comment:
An enthusiastic hint for /cancel ;)

```suggestion
    text = "Send your favorite picture of Covid and use /cancel anytime to leave the demo!"
```